sigma/backends/panther/panther_sdyaml_backend.py

```diff
@@ -71,8 +71,18 @@ def get_key_condition_values(self, cond, state):
 
         return list(rv)
 
-    def convert_condition_and(self, cond: ConditionAND, state: ConversionState) -> Any:
+    def simplify_convert_condition_and(self, cond: ConditionAND, state: ConversionState) -> Any:
         key_cond_values = self.get_key_condition_values(cond, state)
+        simplified = []
+        for key_cond_value in key_cond_values:
+            if key_cond_value.get(self.SDYAML_ALL):
+                simplified.extend(key_cond_value[self.SDYAML_ALL])
+            else:
+                simplified.append(key_cond_value)
+        return simplified
+
+    def convert_condition_and(self, cond: ConditionAND, state: ConversionState) -> Any:

```

Comment:
Do we still need this method? I don't see any references using it but I may have missed something.

Reply:
Yes, it is used in `convert` method of pySigma, we can't drop it